spec/models/mainstream_browse_page_spec.rb

```diff
@@ -209,4 +209,37 @@
       expect(page.lists).to be_an_instance_of(ListSet)
     end
   end
+
+  describe "popular_lists" do
+    let(:top_level_browse_page) { GovukSchemas::Example.find("mainstream_browse_page", example_name: "top_level_page") }
+    let(:second_level_browse_page) { GovukSchemas::Example.find("mainstream_browse_page", example_name: "level_2_page") }
+    let(:root_browse_page) { GovukSchemas::Example.find("mainstream_browse_page", example_name: "root_page") }
+
+    it "passes the content item to PopularListSet" do
```

Comment:
Could this be renamed to `it "passes the content item to PopularListSet on top level browse pages"` to match the other tests.